<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Парсер-Перекодировщик (Практическая ИВМО-04-20)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Путь к файлу:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_browse"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_browse">
          <property name="text">
           <string>Обзор</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Форма</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_parseType">
            <item>
             <property name="text">
              <string>Форма 1 (Кредиты)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Форма 2 (РСБУ)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Кодировка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_codec">
            <item>
             <property name="text">
              <string>windows-1251</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UTF-8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IBM866</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_check">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Проверить подключение к базе</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_parse">
          <property name="text">
           <string>Открыть файл</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="te_dbError"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
